src/plugins/intel_cpu/src/emitters/plugin/x64/jit_load_store_emitters.cpp

```diff
@@ -1006,13 +1010,13 @@ void jit_store_emitter::store_bytes(const Xbyak::Reg64& reg, int offset, int sto
 
     switch (store_size) {
     case 64:
-        h->uni_vmovdqu(addr(0), zmm);
+        h->uni_vmovdqu16(addr(0), zmm);
```

Comment:
Before your changes we call `vmovdqu32` with `zmm` here and `vmovdqu` with `xmm` (they're used in `uni_vmovdqu` pseduo-instructions). Now we use `vmovdqu16` everywhere. Could you please double-check that this is OK behavior here now?

The same comment for `load_emitter` in `load_bytes`

Reply:
As far as I understand, `vmovdqu`, `vmovdqu16`, `vmovdqu32 `and `vmovups `are doing the same thing functionally, i.e. just to move. The different is the time(domain crossing penalties) to move when data type is not correct on old micro-arch, specially for register to register case. So I think it is safe to replace.

Reply:
Agree. 
 Also I've found only `uni_vmovdqu16` in `jit_generator` (8,32,64 are missed) - this is safe for all x64 ISAs, so I think we can leave as it is (since we load just bytes and data type doesn't matter here)
Thank you!